scss/blocks/tabs.scss

```diff
@@ -228,13 +229,17 @@
   background-color: transparent;
   color: var(--colorInkPrimary);
   border-top: 1px solid var(--colorBorderColorSubtle);
-  border-bottom: 1px solid var(--colorBorderColorSubtle);
+  border-bottom: 1px solid transparent;
 }
 
 .accordion-section a.collapsed .accordion-header:hover {
   background-color: var(--colorBackgroundSecondary);
 }
 
+.accordion-section:last-of-type a.collapsed .accordion-header {
+  border-bottom: 0px;
+}
+
```

Comment:
where is this from? I think we should edit Supernova's code as little as we can :)

Reply:
I'm not sure where thats came from, i had no reason to touch any of the accordion scss?